controlplane/src/core/bufservices/PlatformService.ts

```diff
@@ -104,8 +110,14 @@ import {
 import { FederatedGraphSchemaUpdate, OrganizationWebhookService } from '../webhooks/OrganizationWebhookService.js';
 import { OidcRepository } from '../repositories/OidcRepository.js';
 import OidcProvider from '../services/OidcProvider.js';
+import { BlobNotFoundError } from '../blobstorage/index.js';
 import { GraphCompositionRepository } from '../repositories/GraphCompositionRepository.js';
 
+interface PublishedOperationData {
```

Comment:
Please never put types in this file. Either put in a separate file or on types.ts

Reply:
Done